paper-tab.html

```diff
@@ -116,6 +116,18 @@
         Polymer.PaperRippleBehavior
       ],
 
+      /**
+       * Fired when the paper-tab recieves focus.
+       *
+       * @event paper-tab-focus
+       */
+
+      /**
+       * Fired when the paper-tab loses focus.
+       *
+       * @event paper-tab-blur
```

Comment:
Hmm why is the event not just `focus` and `blur`? We don't tend to rename these events when ding retargetting: https://github.com/PolymerElements/iron-behaviors/blob/master/iron-control-state.html#L77